lms/static/scripts/frontend_apps/components/FilePickerFormFields.js

```diff
@@ -33,20 +29,13 @@ export default function FilePickerFormFields({
   content,
   formFields,
   groupSet,
-  ltiLaunchURL,
 }) {
-  /** @type {Record<string,string>} */
-  const extraParams = groupSet ? { group_set: groupSet } : {};
```

Comment:
formFields does now include everything needed for contentitemselection/deeplinking.